frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/utils.ts

```diff
@@ -1,33 +1,34 @@
-import * as _ from 'lodash';
+import { PrometheusHealthHandler, ResourceHealthHandler } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { PrometheusHealthHandler } from '@console/plugin-sdk';
 import { getResiliencyProgress } from '../../../../utils';
+import { WatchCephResource } from '../../../../types';
 
-const CephHealthStatus = [
-  {
+const CephHealthStatus = {
+  HEALTH_OK: {
     state: HealthState.OK,
   },
-  {
+  HEALTH_WARN: {
     state: HealthState.WARNING,
   },
-  {
+  HEALTH_ERR: {
```

Comment:
looking at this change...are these `CephHealthStatus` keys supposed to correspond with https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts#L29 ?

Reply:
Actually it has to correspond to ceph health which has 3 fixed statuses
```
HEALTH_OK
HEALTH_WARN
HEALTH_ERR
```
I did a mistake in initial PR by putting `HEALTH_ERR` as `HEALTH_ERROR` .